### PR TITLE
Sorting tasks by category, then group.

### PR DESCRIPTION
**Pull request by scalvert**
Dependent Tasks:

- [ ] Add default categories to the docs site.

### CODE REVIEW

File: packages/cli/src/task-result-comparator.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { TaskClassification, TaskResult } from '@checkup/core';
+
+const DEFAULT_CATEGORIES: Record<string, number> = {
+  metrics: 6,
```
Comment (carakessler) [inline packages/cli/src/task-result-comparator.ts:4]: I think we would want this prioritization to be configurable, no?  

also - if we are going to "rate"  what we consider to be the "default categories", we definitely need to document this somewhere so people are _somewhat_ encouraged to adhere to default categories

**Comment by carakessler** [comment]
Can we add tests for this por favor?

**Comment by scalvert** [comment]
Already on it! I wouldn't leave home without tests :)

File: packages/cli/src/task-result-comparator.ts (hunk 1 above)
Comment (scalvert) [reply packages/cli/src/task-result-comparator.ts:4]: I don't think we want default categories configurable. I do believe we should document the default categories. That can be added to the docs site.

**Comment by scalvert** [comment]
Added tests @carakessler. I think this is GTG - I will add documentation after this lands.

**Pull request merged**